hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1307,13 +1307,77 @@
               "chip_sw_hmac_enc_jitter_en",
               "chip_sw_keymgr_key_derivation_jitter_en",
               "chip_sw_kmac_mode_kmac_jitter_en",
-              "chip_sw_sram_ctrl_main_scrambled_access_jitter_en"
-             ]
+              "chip_sw_sram_ctrl_main_scrambled_access_jitter_en"]
     }
     {
       name: xcelium_ci
       tests: ["chip_plic_all_irqs"]
       reseed: 1
     }
+    {
+      name: xcelium_ci_0
+      tests: ["chip_plic_all_irqs"]
+              // TODO(#14532): enable these tests slowly to ensure no CI breakages.
+              // "chip_sw_clkmgr_jitter",
```

Comment:
If these are not run, you could uncomment these, right?

Reply:
Please see my above comment. 

To summarize, in order to slowly roll out these tests to CI (so as not to break or overload CI) the following needs to happen in the exact order below:
1. add more regression suites with a single test that is known to already run successfully in CI (what this PR does; the "chip_plic_all_irqs" test is known to work already as it is already a blocking private CI test, and the tests that will eventually take its place are in comments)
2. update private CI job config file (which is in a separate repo) to run these 5 regression suites as 5 separate parallel jobs (or maybe just start with 3 to slowly roll out)
3. create several follow up PRs in this (main) repo that slowly uncomment tests in these regressions so we can see if: 1) they actually run successfully (they should since they run in the nightly fine for the time being) and 2) how they impact CI run time _**before**_ merging them.

What we want to avoid is adding these 5 regressions in this repo, and then updating the private CI job config file in the other repo to make all 5 regressions live immediately and have CI blowup because either: 1) there are not enough private CI VMs to handle the increased workload, or 2) some of these tests all of a sudden are broken in the nightly regression so they break CI, causing us to have to roll back.

Hopefully that makes sense.